### ISSUE TITLE
Inconsistency for decreasing loss

### ISSUE THREAD

**Issue by BenjiKCF** [fchollet/keras#6004]
- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

Hello everyone, I have been using this script [Python script](https://github.com/etai83/lstm_stock_prediction/blob/master/.ipynb_checkpoints/GOOGLE%20stock%20prediction-checkpoint.ipynb) 
The problem is that I am unable to produce similar result everytime. 
Sometimes, I can produce a similar result (loss of 0.3~ within 500 epochs) but sometimes I still get a loss of 3.x after 1500epochs. I am not sure if this is a bug or this is because the algorithm just stuck at a minimum local.

**Comment by BenjiKCF** [comment]
By applying normalization, the problem will be solved
